config/settings.py

```diff
@@ -133,3 +134,7 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = "/static/"
+
+CLIENT_DEV_URL = "http://localhost:3000"
+
+DEVELOPMENT_MODE = True
```

Comment:
there's already a Django setting for this, `settings.DEBUG`, I recommend just using it.